trade-finance-manager-ui/component-tests/utilisation-reports/add-payment.component-test.js

```diff
@@ -40,4 +40,260 @@ describe(page, () => {
     wrapper.expectText('[data-cy="selected-reported-fees-details-table"]').toContain('JPY 3');
     wrapper.expectText('[data-cy="selected-reported-fees-details-table"]').toContain('JPY 1');
   });
+
+  it('should display a currency option for each currency', () => {
+    // Arrange
+    const addPaymentViewModel = anAddPaymentViewModel();
+    const wrapper = render(addPaymentViewModel);
+
+    // Assert
+    wrapper.expectElement('input[type="radio"][value="GBP"]').toExist();
+    wrapper.expectElement('label:contains("GBP")').toExist();
+    wrapper.expectElement('input[type="radio"][value="JPY"]').toExist();
+    wrapper.expectElement('label:contains("JPY")').toExist();
+    wrapper.expectElement('input[type="radio"][value="USD"]').toExist();
+    wrapper.expectElement('label:contains("USD")').toExist();
+    wrapper.expectElement('input[type="radio"][value="EUR"]').toExist();
+    wrapper.expectElement('label:contains("EUR")').toExist();
+  });
+
+  it('should not initialise payment currency when no value given', () => {
+    // Arrange
+    const addPaymentViewModel = anAddPaymentViewModel();
+    addPaymentViewModel.formValues.paymentCurrency = undefined;
+    const wrapper = render(addPaymentViewModel);
+
+    // Assert
+    wrapper.expectInput('input[type="radio"][value="GBP"]').notToBeChecked();
+    wrapper.expectInput('input[type="radio"][value="JPY"]').notToBeChecked();
+    wrapper.expectInput('input[type="radio"][value="EUR"]').notToBeChecked();
+    wrapper.expectInput('input[type="radio"][value="USD"]').notToBeChecked();
+  });
+
+  it('should initialise payment currency when value given', () => {
+    // Arrange
+    const addPaymentViewModel = anAddPaymentViewModel();
+    addPaymentViewModel.formValues.paymentCurrency = 'USD';
+    const wrapper = render(addPaymentViewModel);
+
+    // Assert
+    wrapper.expectInput('input[type="radio"][value="USD"]').toBeChecked();
+  });
+
+  it('should display error message when there is an error with the payment currency', () => {
+    const addPaymentViewModel = anAddPaymentViewModel();
+    addPaymentViewModel.errors.paymentCurrencyErrorMessage = 'Please, select a currency';
```

Comment:
For this comment and those below, I think I've realised this is because it's just a component test, so the actual content of the error message doesn't matter. That being said, I think it might be nice to have the error messages used be consistent with those we'd actually expect to see :)

Reply:
I'm not convinced it matters. For at least one field there is more than one error message and I'm not setting field values or anything, I almost think best to have something different to the standard message to emphasise that the component should just is display the error message we pass in?

Reply:
It doesn't matter really - it was just the first thing I saw on the PR and confused me because it didn't line up with the ticket. Happy to resolve :)